lib/build.sh

```diff
@@ -142,6 +142,15 @@ install_or_reuse_yarn() {
   fi
 }
 
+set_node_env() {
+  local layer_dir=$1
+
+  mkdir -p "${layer_dir}/env.launch"
+  if [[ ! -s "${layer_dir}/env.launch/NODE_ENV" ]]; then
+    echo "production" >> "${layer_dir}/env.launch/NODE_ENV"
```

Comment:
The only thing is we should use `NODE_ENV` if it's set, and then use `production` as default. (I'll make a follow up commit since Lillian's on vacation)